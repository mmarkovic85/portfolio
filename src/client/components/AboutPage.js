import React from "react";

const AboutPage = () => (
  <h1>About page placeholder</h1>
);

export {
  AboutPage as default
};
