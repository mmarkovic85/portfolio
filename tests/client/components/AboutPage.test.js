import React from "react";
import renderer from "react-test-renderer";
import AboutPage from "../../../src/client/components/AboutPage";

test("Should render AboutPage component correctly", () => {
  const rendered = renderer.create(
    <AboutPage />
  );

  expect(rendered.toJSON()).toMatchSnapshot();
});
