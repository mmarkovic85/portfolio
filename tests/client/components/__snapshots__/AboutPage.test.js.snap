// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render AboutPage component correctly 1`] = `
<h1>
  About page placeholder
</h1>
`;
